fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonUtils.xtend

```diff
@@ -135,6 +135,24 @@ class PythonUtils {
     		return "";
     	}
     }
+    def commentMultiLinesDescriptionStr(String str){
+    	if (str != null)
+    	{
+    		if (!str.empty)
+    		{
+				return "" + myReplaceAll(str,"\n","\"\n        \"");
+    		}
+    		else
+    		{
+    			return "" ;
+    		}
+    	}
+    	else
+    	{
+    		return "" ;
+    	}
+    }
+        
```

Comment:
This result is much better, but could be a little better still.  Currently if gives:
```python
     @command(
        dtype_in='DevBoolean',
        doc_in="Any boolean value"
        "Description of input argument in multiple lines."
        "This is input argument for command.",
        dtype_out='DevBoolean',
        doc_out="Echo of the argin value"
        "Description of output argument in multiple lines."
        "This is ouput argument for command.",
    )
```
It would be nicer if all the opening quotes were aligned:
```python
     @command(
        dtype_in='DevBoolean',
        doc_in="Any boolean value"
               "Description of input argument in multiple lines."
               "This is input argument for command.",
        dtype_out='DevBoolean',
        doc_out="Echo of the argin value"
                "Description of output argument in multiple lines."
                "This is ouput argument for command.",
    )
```

Reply:
Aligned the comments in one line. For output argument description, created separate method. I have attached the newly generated code.